src/redux/reducers/app.js

```diff
@@ -1,15 +1,26 @@
 import { handleActions } from 'redux-actions';
 import actionTypes from '../actionTypes';
+import Cookies from 'universal-cookie';
+
+const cookies = new Cookies();
 
 const defaultState = {
-  name: '',
+  userName: '',
   email: '',
   accessToken: '',
   uuid: '',
   isAdmin: false,
   apiKeys: {},
 };
 
+const cookiesAppValues = {
+  email: cookies.get('emailId'),
```

Comment:
Try using `cookies.getAll` and then assigning individual values. Multiple `get` can be slower.
https://github.com/reactivestack/cookies/tree/master/packages/universal-cookie#getalloptions

Reply:
@akshatnitd I have changes the names of cookies to match the state keys, made it easier to destructure from cookies.getAll(). hope it isn't a problem. :sweat_smile: